Project.toml

```diff
@@ -1,14 +1,14 @@
 name = "FFMPEG"
 uuid = "c87230d0-a227-11e9-1b43-d7ebe4e7570a"
 authors = ["SimonDanisch <sdanisch@gmail.com>"]
-version = "0.4.1"
+version = "1.0.0"
```

Comment:
I don't think this needs to be a breaking change.

Reply:
Or is it so because of the minimum julia version bump?

Reply:
That's not a breaking change, it only warrants a change in the second non-zero number.